README.md

```diff
@@ -50,13 +50,16 @@ epp-client = "0.3"
 
 You can create a mut variable of type `EppClient` with the domain registry config.
 
+```rust
```

Comment:
Could you remove this line? Seems to be a duplicate start of the code block. It's repeated on Line 54.